package.json

```diff
@@ -58,14 +59,15 @@
   "devDependencies": {
     "ava": "^0.15.0",
     "babel-eslint": "^5.0.0-beta6",
+    "babel-plugin-rewire": "^1.0.0-rc-4",
```

Comment:
Is there a reason this is in both deps and devDeps? Also, the version ranges do not match (this one starts with a `^`)